proto/README.md

```diff
@@ -20,7 +20,36 @@ The canonical Protobuf definitions live in the `proto` folder of the relevant
 release branch of CometBFT. These definitions are published to the [Buf
 registry][buf] for integrators' convenience.
 
-## Why does CometBFT use `tendermint` Protobuf definitions?
+The Protobuf files are organized under two domains: `cometbft` and `tendermint`.
+The `cometbft.*` packages use version suffixes to let application developers
+target versions of the protocols as they have evolved between CometBFT releases.
+
+## Which CometBFT release does each package belong to?
+
+By the 1.0.0 release, the entire set of Protobuf definitions used by CometBFT
+is published in packages suffixed with `.v1`. Earlier revisions of the
+definitions, where they differed, are provided alongside in `.v1beta`_N_
+packages. The correspondence between package suffixes and releases is as follows:
+
+| Domain          | 0.34      | 0.37      | 0.38      | 1.0  |
+|-----------------|-----------|-----------|-----------|------|
+| `abci`          | `v1beta1` | `v1beta2` | `v1beta3` | `v1` |
+| `blocksync`     |           | `v1beta1` | `v1`      | same |
+| `consensus`     | `v1beta1` | same      | same      | `v1` |
+| `crypto`        | `v1`      | same      | same      | same |
+| `libs/bits`     | `v1`      | same      | same      | same |
+| `mempool`       | `v1`      | same      | same      | same |
+| `p2p`           | `v1`      | same      | same      | same |
+| `privval`       | `v1beta1` | same      | `v1`      | same |
+| `rpc/grpc`[^1]  | `v1beta1` | `v1beta2` | `v1beta3` |      |
+| `state`         | `v1beta1` | `v1beta2` | `v1beta3` | `v1` |
+| `statesync`     | `v1`      | same      | same      | same |
+| `types`         | `v1beta1` | `v1beta2` | `v1`      | same |
+| `version`       | `v1`      | same      | same      | same |
+
+[^1]: Retired in 1.0
+
+## Why does CometBFT provide `tendermint` Protobuf definitions?
```

Comment:
Is the assumption that the packages were just renamed ? Like if someone was using the `tendermint.*` packages before, and want to upgrade, they would just need to rename to `cometbft.*` ? like they are 1:1 compatible (in structure)? If so, maybe a small blurb of text explaining (and assuring that) would be beneficial I'd think.

Reply:
They were renamed _and_ versioned.